src/scripts/start_valheim.sh

```diff
@@ -63,6 +63,56 @@ cp /home/steam/steamcmd/linux64/steamclient.so /home/steam/valheim/linux64/
 log "Initializing Variables...."
 odin configure || exit 1
 
+log "Checking for TYPE flag"
+export TYPE="${TYPE:="vanilla"}"
```

Comment:
It may be somewhere and I just missed it, but it looks like there's never a check for a valid `$TYPE` so someone could accidentally misspell the name of the mod framework and end up in a vanilla world on accident.

Reply:
Expected, KIS approach :P